tests/SerializerPhp81Test.php

```diff
@@ -183,6 +183,20 @@ enum SerializerScopedBackedEnum: string {
     );
 })->with('serializers');
 
+test('readonly properties declared in parent', function () {
```

Comment:
can you add a bunch more tests into this? 

- for things like properties values, of children an parent classes, etc.
- etc

Reply:
I am not sure I understood that to test here🤔
I can only think of this example
```php
class Parent
{
    public function __construct(
        public readonly int $property = 1,
    ) {}
}

class Child extends Parent 
{
    public function __construct(
        public readonly int $property = 2,
    ) {}
}
```
But this already works without fix because property is redeclared in Child so it has the same scope.
Or maybe you mean tests should check that if we created Child with property 100 it should stay 100 after serialization and deserialization?

Reply:
> Or maybe you mean tests should check that if we created Child with property 100 it should stay 100 after serialization and deserialization?

Yes. Exactly that.